lua/solarized-osaka/theme.lua

```diff
@@ -496,6 +496,7 @@ function M.setup()
     NeoTreeNormal = { fg = c.base00, bg = c.bg_sidebar },
     NeoTreeNormalNC = { fg = c.base00, bg = c.bg_sidebar },
     NeoTreeDimText = { fg = c.base01 },
+    NeoTreeMessage = { fg = util.lighten(c.base02, 0.8) },
```

Comment:
please don't use `util.lighten`. Can you pick a color from the palette instead?

Reply:
can i use `util.darken` to darken at least?
Otherwise the "hidden items" blends in with the file names color and using one hue below makes it blend with the hidden files being shown.
If not i will just push with color from the palette that is decent enough.

Reply:
No, because the concept of this theme is ditching these utility functions. The color palette is a constraint to keep the color consistency.

Reply:
I saw it being used for a variety of things there hence why i used it.
Nevertheless pushed with a color from a palette.